From 8e318d7a99717e7c4b79fdd18c55a02a34ffc6e5 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 10 Jul 2018 16:18:06 +0300
Subject: [PATCH] board: habana: sync the firmware binaries

Sync the firmware binaries before the habalabs driver build and install.
This way the jenkins job is actually tests the lates binaries.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_load_firmware b/board/habanalabs/common/remote_load_firmware
index 42f6835..3d56689 100755
--- a/board/habanalabs/common/remote_load_firmware
+++ b/board/habanalabs/common/remote_load_firmware
@@ -3,6 +3,7 @@
 host=$1
 success_str="1000"
 result=~/remote_load_firmware.result
+firm_dir=/lib/firmware/habanalabs/dali
 rm -f $result
 
 for ((i=100; i>0; i--)); do
@@ -15,6 +16,8 @@
 		echo ssh repo sync status: $pipestat
 		[ $pipestat -ne 0 ] && continue;
 
+		sshpass -p123456 scp ${firm_dir}/* root@${host}:${firm_dir}/
+
 		ssh -t $host bash -ic 'bi_habanalabs' | tee -a $result
 		success=`grep -cE "^$success_str" $result`
 		if [ $success -ge 1 ]; then
